exercises/exercise_01/README.md

```diff
@@ -65,7 +65,7 @@ This runs the model with the array `[0.0, 1.0, 2.0, 3.0, 4.0]` should produce th
 ### Task 1
 
 ```python
-# FPTLIB-TODO
+# FTORCH-TODO
```

Comment:
Maybe just TODO? For consistency with what's in the Fortran exercises? I guess we might consider switching away from this notation in FTorch itself, too.

Reply:
Yes, that did occur to me.